src/java.base/share/classes/java/lang/runtime/SwitchBootstraps.java

```diff
@@ -160,6 +165,11 @@ private static int doTypeSwitch(Object target, int startIndex, Object[] labels)
                 } else if (target instanceof Character input && constant.intValue() == input.charValue()) {
                     return i;
                 }
+            } else if (label instanceof EnumDesc<?> enumDesc) {
+                if (target.getClass().isEnum() &&
+                    ((Enum<?>) target).describeConstable().stream().anyMatch(d -> d.equals(enumDesc))) {
```

Comment:
probably a matter of style but isn't it an overkill to use streams here? there will be always only one element right?

Reply:
Technically, one element, but the type is optional, so formally could be no elements. I could use `.get()`, but that feels a bit ugly and against optional?

Reply:
yep I know, up to you I'm not strong on this, I guess I was thinking about performance but probably not using streams won't make a difference in this case